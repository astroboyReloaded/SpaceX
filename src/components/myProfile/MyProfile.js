import MyMissions from './MyMissions';
import MyRockets from './MyRockets';
import profile from '../../styles/myProfile.module.css';

const MyProfile = () => (
  <div className={profile.mainContainer}>
    <MyMissions />
    <MyRockets />
  </div>
);

export default MyProfile;
