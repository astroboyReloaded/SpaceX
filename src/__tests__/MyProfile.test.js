import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/Store';
import MyProfile from '../components/myProfile/MyProfile';

it('renders My Profile', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <Routes>
            <Route path="/profile" element={<MyProfile />} />
          </Routes>
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
