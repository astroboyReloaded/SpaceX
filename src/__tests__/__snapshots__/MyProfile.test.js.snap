// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders My Profile 1`] = `null`;
